opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -378,9 +383,10 @@ def use_span(self, span: 'Span') -> typing.Iterator[None]:
         yield
 
 
-_TRACER: typing.Optional[Tracer] = None
-_TRACER_FACTORY: typing.Optional[
-    typing.Callable[[typing.Type[Tracer]], typing.Optional[Tracer]]] = None
+FactoryType = Callable[[Type[Tracer]], Optional[Tracer]]
```

Comment:
Please add a comment referencing https://github.com/open-telemetry/opentelemetry-python/pull/95/files#diff-27dca41b0ed750373d4340152e76a2d2R84 (line 84 in loader.py) and use a name aligned to that.

Reply:
It's done in e2c58662f34fe688705560757d008bd2831eb168.